scripts/suse_functions.sh

```diff
@@ -0,0 +1,35 @@
+OBS_DIR=${OBS_DIR:-~/obs}
+source_dir="$(pwd)"
+package_name="$(basename $(pwd))"
+prepare_obs() {
+    project_name="$1"
+
+    tempdir="$(mktemp -d)"
+    echo "Packaging sources w/ tito"
+    tito build --srpm --test -o "$tempdir"
+
+    if [ ! -d ~/obs ]; then
+        mkdir ~/obs
+    fi
+    pushd ~/obs
+
+    if [ ! -d "$project_name" ]; then
+        osc co "$project_name"
+        pushd "$project_name"
+    else
+        pushd "$project_name"
+        echo "Ensuring $project_name is up-to-date."
+        osc update
+    fi
+
+    pushd "$package_name"
+    echo "Removing any existing sources"
+    osc rm *
```

Comment:
@awood, `osc` acts much like `git` does, and in this case, unfortunately, one needs to be inside the package checkout to remove files (`osc rm ${package_name}/*` won't work). It does however refuse to remove files that aren't part of an OBS checkout:

```console
$ cd /tmp; touch test; osc rm test
Error: "/tmp" is not an osc package working copy.
$ test -f /tmp/test && echo 'okay'
okay
```